hack/Containerfile.local

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/centos-bootc/centos-bootc:stream9
+
+COPY bin/rpm/flightctl-agent-*.rpm /tmp/
+RUN dnf install -y /tmp/flightctl-agent-*.rpm && \
+    systemctl enable flightctl-agent.service
+
+RUN useradd -ms /bin/bash redhat && \
+    echo "redhat:redhat" | chpasswd && \
+    echo "redhat ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers
+
+## Add your flightctl configuration and certificates
+ADD hack/config.yaml /etc/flightctl/
+ADD hack/ca.crt /etc/flightctl/certs/
+ADD hack/client-enrollment.* /etc/flightctl/certs/
```

Comment:
```suggestion
ADD bin/agent/etc/flightctl/config.yaml /etc/flightctl/
ADD bin/agent/etc/flightctl/certs/* /etc/flightctl/certs/
```

Reply:
small comment to make it work with: https://github.com/flightctl/flightctl/pull/186